src/components/InteractiveList/InteractiveListInt.part.tsx

```diff
@@ -60,14 +61,15 @@ const ListWrapper = styled('ul')<InteractiveListWrapperProps>(
 
 const ListItem = styled.li<ListItemProps>(
   themed(
-    ({ hovered, theme: { ui3, text2 } }) => css`
+    ({ hovered, theme: { ui3, text2 }, disabled }) => css`
       ${getFontStyle({ size: 'medium' })}
       
       background: ${hovered ? ui3 : transparent};
       color: ${text2};
       list-style: none;
       box-sizing: border-box;
-      cursor: pointer;
+      opacity: ${disabled ? '0.5' : '1.0'};
```

Comment:
`opacity` is a very expensive attribute to render in css. I would use a background-color or font color instead.

Reply:
I've used opacity here to match the styling of [Checkbox](https://github.com/ZEISS/precise-ui/blob/develop/src/components/Checkbox/index.tsx#L41) which also uses opacity.